inc/class-cachify-interface.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Interface for APC based caching storage classes.
+ *
+ * @package Cachify
+ */
+
+/* Quit */
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Interface_Cachify_Storage
+ */
+interface Cachify_Interface {
+
+	/**
+	 * Availability check
+	 *
+	 * @return bool TRUE if available
+	 */
+	public static function is_available();
+
+	/**
+	 * Caching method as string
+	 *
+	 * @return  string Caching method
+	 */
+	public static function stringify_method();
```

Comment:
I do dislike this method name and would have called it something as simple as `get_name()`. But that should probably be done separately as this is only a copy of the pre-existing implementation.

Reply:
Yes I guess we can refactor the name after this PR.